system/lib/compiler-rt/wasm_eh_tags.S

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
I think it maybe makes more sense to define `__cpp_exception` in libc++abi. IIRC that's where we define the functions that the compiler emits calls to such as `__cxa_throw`.
Slightly annoying that we'd be separating  `__cpp_exception` from `__c_longjmp` but I think its' the right layering.
Speaking of which, where does `__c_longjmp` belong? Do we define `longjmp` in compiler-rt or libc?

Reply:
Is the latter file in compiler-rt because `setThrew` is shared between EH and SJLJ?

Reply:
Ah right, `setThrew` is also used in Emscripten SjLj. Yeah I think that's the reason.

Reply:
Where are we putting tag symbols libcxxabi or compiler? https://github.com/llvm/llvm-project/pull/159143 puts both in compiler-rt but this puts `__cpp_exceptions` in libc++abi, so..

Personally I think putting both in compiler-rt is fine, given that these are things we never get to upstream to the upstream libc++abi anyway.

Reply:
I agree we should match https://github.com/llvm/llvm-project/pull/159143 once it lands.